Packs/JoeSecurity/Playbooks/playbook-Detonate_File_From_URL_-_JoeSecurity.yml

```diff
@@ -362,3 +362,5 @@ outputs:
 - contextPath: DBotScore.Score
   description: The actual score (only in case of report type=json).
   type: number
+tests:
```

Comment:
pretty sure we have a test for this

Reply:
I see that `detonatefile_-_joesecurity` has a test = `Detonate File - Generic Test`

should this playbook use the same test?

Reply:
i think that your filter is not catching it or something, need to make sure we are in the clear here